docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -336,6 +337,26 @@ protected function replaceFidsWithFileContents(WebformInterface $webform, array
     }
   }
 
+  /**
+   * Add reference to files that have been removed by virus scanning.
+   *
+   * @param \Drupal\webform\WebformInterface $webform
+   *   The webform the submission belongs to.
+   * @param array $submissionValues
+   *   The values submitted.
+   */
+  protected function includeReferenceToRemovedFiles(WebformInterface $webform, array &$submissionValues) {
+    $fileAttachmentElementNames = $this->getFileAttachmentElementsOnWebform($webform);
+    if ($fileAttachmentElementNames) {
+      foreach ($fileAttachmentElementNames as $fileAttachmentElementName) {
+        if (isset($submissionValues[$fileAttachmentElementName])) {
+          $attachments = $this->getRemovedFiles($submissionValues[$fileAttachmentElementName]);
+          $submissionValues['removed_files'] = $attachments;
```

Comment:
Should we rename the $attachments variable to $deletedAttachments or $removedAttachements?